src/VirtualList/__tests__/VirtualList-test.js

```diff
@@ -131,40 +133,12 @@ describe('VirtualList', function () {
     var itemHeight = 200;
     var itemCount = 20;
 
-    it('shows items that are in the viewport', function () {
```

Comment:
Meaning, I think we should still test that the right amount of items are in view